apis/codegen_test.go

```diff
@@ -30,9 +29,7 @@ func TestCodeGeneration(t *testing.T) {
 
 	// check that the returned byte thing is correct
 	generated := strings.Trim(string(source), "\n\r\t ")
-	desired, err := getDesiredOutput("services_test.go")
-
-	assert.NoError(err, fmt.Sprintf("error getting desired output: %s", err))
+	desired := getDesiredOutput("services_test.go")
 	assert.Equal(generated, desired, "generated code doesn't match desired code")
```

Comment:
Here, the format should be `assert.Equal(expected, actual, msg)` :) https://godoc.org/github.com/stretchr/testify/require#Assertions.Equal